lib/api/api_test.go

```diff
@@ -119,7 +119,7 @@ func TestStopAfterBrokenConfig(t *testing.T) {
 	defer os.Remove(token)
 	srv.started = make(chan string)
 
-	sup := suture.New("test", util.Spec())
+	sup := suture.New("test", util.Spec(func(e suture.Event) { l.Debugln(e) }))
```

Comment:
I think this would be less ugly if we had:
```
util.SpecWithDebugLogger(l)
util.SpecWithInfoLogger(l)
```
or
```
util.SpecWithLogger(l, log.INFO_LEVEL)
```